cmake/pkgconfig.in

```diff
@@ -6,6 +6,6 @@ Name: libOSRM
 Description: Project OSRM library
 Version: v@OSRM_VERSION_MAJOR@.@OSRM_VERSION_MINOR@.@OSRM_VERSION_PATCH@
 Requires:
-Libs: -L${libdir} -lOSRM
+Libs: -L${libdir} -losrm
```

Comment:
Yes but the `Libs` section should only include the provided libs. Everything else should go in `Libs.private` that is used by `pkg-config` when you specify the `--static` flag. Sadly cmake does not play well with that. (we would need to detect if `libosrm` was compiled statically or dynamic and use the corresponding library flags in `FindOSRM.cmake`)